src/objects.js

```diff
@@ -1,41 +1,79 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+  name: name,  
+  age: age
+  }
+  return person;
```

Comment:
```suggestion
  const person = {
    name,
    age,
  }
  return person;
```
Indentation, plus this shorthand can be used to make code a little cleaner